src/components/PublicPursuances/PublicPursuances.css

```diff
@@ -0,0 +1,72 @@
+.order-by label {
+  margin-right: 10px;
+  color: white;
+  font-weight: normal;
+}
+
+.custom-dropdown {
```

Comment:
This dropdown doesn't seem to match the style of other dropdowns, but still looks good IMO.

![screenshot_20180706_200128](https://user-images.githubusercontent.com/35820227/42395913-0d2b44c8-814f-11e8-9d3c-3f8fea03a131.png)

As an aside, I couldn't find a way to navigate to pursuances/all normally.